.travis.yml

```diff
@@ -6,8 +6,7 @@ services:
 install: skip
 branches:
   only:
-  - master
```

Comment:
Please remove this unrelated changes.

Reply:
Yeah, sure, it anyway will force me to do that because of a merge conflict :)